english/PortBlair's Emerald blue/PortBlairVoyage.md

```diff
@@ -0,0 +1,15 @@
+# I landed on Port Blair on **26th Jan** 2008
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.